lib/modules/platform/github/index.ts

```diff
@@ -1177,23 +1176,22 @@ export async function setBranchStatus({
 
 /* istanbul ignore next */
 async function getIssues(): Promise<Issue[]> {
-  const result = await githubApi.queryRepoField<Issue>(
-    getIssuesQuery,
-    'issues',
-    {
-      variables: {
-        owner: config.repositoryOwner,
-        name: config.repositoryName,
-        user: config.renovateUsername,
-      },
-    },
-  );
-
-  logger.debug(`Retrieved ${result.length} issues`);
-  return result.map((issue) => ({
-    ...issue,
+  const result: Issue[] = (
+    await githubApi.getJson<Issue[]>(
```

Comment:
Why don't use we a schema here?

Reply:
yes, we should not exclude ourselves from schema 🙃